src/fs/dvfs/dvfs_str.c

```diff
@@ -26,32 +26,36 @@ const char *dvfs_last_link(const char *path) {
 
 	len = strlen(path);
 	r = path + len - 1;
-	/* Miss traling slashes */
-	while (*r == '/' && r > path)
+	/* Miss trailing slashes */
+	while (*r == '/' && r > path) {
 		r--;
+	}
 
 	/* Find previous slash or begin of path */
 	l = r - 1;
-	while (*l != '/' && l > path)
+	while (*l != '/' && l > path) {
 		l--;
+	}
 
 	return l;
 }
 
-
+#if 0
```

Comment:
I think that this function is trivial enough to be safely removed.

Reply:
done.
Rm the whole file